tests/fork/aave-oracle.spec.ts

```diff
@@ -0,0 +1,244 @@
+import { ONE_ADDRESS, waitForTx, ZERO_ADDRESS } from "../../helpers";
+import { makeSuite, TestEnv } from "../utils/make-suite";
+import { expect } from "chai";
+
+makeSuite('AaveOracle', (testEnv: TestEnv) => {
+  const assetPrice = '1000000000000000000';
+  const CALLER_NOT_RISK_OR_POOL_ADMIN = '4';
+  const CALLER_NOT_ASSET_LISTING_OR_POOL_ADMIN = '5';
+  const INCONSISTENT_PARAMS_LENGTH = '76';
```

Comment:
I think ideally you extract these errors from the compiled code, just for inmutability, As is, if value changes in source code tests would fail although reason is the same